utils/pcf_compare.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```

Comment:
I'm unclear why you would want to do what this script does?

Reply:
It compares two PCF files. It is used in QuickLogic tests to verify whether design constraints were correctly applied during the toolchain flow. `fasm2bels` for QuickLogic PP3 architecture can write PCF with actual IO locations as encoded in the bitstream. This is verified against a design.